src/components/search-result-list-item/search-result-list-item.styles.scss

```diff
@@ -0,0 +1,27 @@
+.searchResult {
+  background: #f2f2f2;
+  border-left: 46px solid #c2d0de;
+  padding: 20px 25px;
+
+  &.cyan {
+    border-color: #c3ece9;
+  }
+  &.blue {
+    border-color: #a6c6f0;
+  }
+
+  .searchResult__date {
+    text-transform: uppercase;
+  }
+
+  .searchResult__title {
+    font-size: 1.3rem;
+    color: #000;
+    margin: 0.8rem 0;
+  }
+
+  .searchResult__moreLink {
+    margin-top: 15px;
+    text-decoration: underline;
+  }
```

Comment:
We should add these to the `<a>` element, otherwise we will have different colors for the link and the `underline`